packages/modules/viewer-react/src/components/iModel/IModelLoader.tsx

```diff
@@ -39,19 +40,15 @@ const IModelLoader = React.memo((viewerProps: ModelLoaderProps) => {
     viewCreatorOptions,
     blankConnectionViewState,
     uiProviders,
-    theme,
-    backstageItems, // eslint-disable-line deprecation/deprecation
     loadingComponent,
+    selectionStorage,
   } = viewerProps;
   const { error, connection } = useConnection(viewerProps);
 
   const providers = useMemo<UiItemsProvider[]>(() => {
     const providers = [...(uiProviders || [])];
-    if (backstageItems?.length) {
-      providers.push(new BackstageItemsProvider(backstageItems));
-    }
     return providers;
-  }, [uiProviders, backstageItems]);
+  }, [uiProviders]);
```

Comment:
Not sure if we care, but `uiProviders` being an array this useMemo will always re-evaluate, no?

Reply:
Ah yes, thanks.